packages/reference/cypress/integration/search/price-filter.js

```diff
@@ -27,7 +27,6 @@ describe('Search', () => {
       }).as('algoliaSearch')
 
       // Wait for page to load
-      cy.wait('@getMenus')
```

Comment:
I think I added this as a global stub in Cypress inside of `global-stubs.js`, I'd be intrigued to see if we can remove it from there and everything passes 🤔

Reply:
worth seeing - i'll give it a go.  Technically this request can still get made, so we maybe shouldn't remove the stub altogether, but I can certianly see if it passes without

Reply:
yup - passes without that stub \o/

Reply:
Nice, was curious more than anything :D as you say leave it in it won't harm anyhow!